script.js

```diff
@@ -69,14 +85,60 @@ function shuffleCards(array) {
 let shuffledCardsArray = shuffleCards(cardObjectsArray);
 let cardDivArray = document.querySelectorAll('.card');
 
-for(let i = 0; i < cardDivArray.length; i++) {
+for (let i = 0; i < cardDivArray.length; i++) {
     cardDivArray[i].innerHTML = '<img class="hamster" src="' + shuffledCardsArray[i].frontOfCard + '">'
-        + '<img class="sloth" src="images/cardBack.jpg">';
+        + '<img class="sloth" data-pair ="' + shuffledCardsArray[i].pairID + '" src= "images/cardBack.jpg">';
+
 }
 
+let clickCounter = 0
+let turnCounter = 0
+let firstClickPair = "start"
+let secondClickPair = "start"
 let slothClickArray = document.querySelectorAll('.sloth');
-slothClickArray.forEach(sloth => {
-    sloth.addEventListener('click', (e)=>{
-        sloth.classList.add('hide');
-    })
-})
+let matchedPairs = 0
+
+function addClickListener(onClick) {
```

Comment:
rename to `addCardClickListeners()` or something like that - it is adding multiple listeners and specifically the click listeners for the cards, also because of the scope of named functions you don't have to pass the function as a parameter, it will work anyway

Reply:
done - harry